ceopardy.py

```diff
@@ -33,6 +46,60 @@ def start():
 def gameboard():
     return render_template("gameboard.html")
 
+@app.route('/host')
+def host():
+    return render_template('host.html')
+
+@app.route('/player')
+def player():
+    return render_template('player.html')
+
+@app.route('/viewer')
+def viewer():
+    return render_template('viewer.html')
+
+@app.route('/comand', defaults={'path': 'debug'})
+@app.route('/comand/<path:path>', methods=['POST'])
+def command(path):
+    print(request.form, file=sys.stderr)
+    result = {}
+    result["result"] = "ok"
+    return flask.jsonify(**result)
+
+@socketio.on('click')
+def handle_click(data):
+    print('received data: ' + data["id"], file=sys.stderr)
+    global controller
+    match = re.match("c([0-9]+)q([0-9]+)", data["id"])
+    if match is not None:
+        items = match.groups()
+        column = int(items[0])
+        row = int(items[1])
+        controller.set_question_solved(column, row, True)
+        state = controller.dictionize_questions_solved()
+        emit("update-board", state, broadcast=True)
+        emit("show-overlay", "Test!", broadcast=True)
+
+@socketio.on('unclick')
+def handle_click(data):
+    emit("hide-overlay", broadcast=True)
+
+@socketio.on('roulette')
+def handle_roulette():
```

Comment:
Would add a comment: doing this server-side ensures that all viewers sees the same random pattern before the final team is selected